x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/create_rules_route.ts

```diff
@@ -78,9 +79,15 @@ export const createRulesRoute = (
 
         // This will create the endpoint list if it does not exist yet
         await ctx.lists?.getExceptionListClient().createEndpointList();
+        const exceptionLists = checkDefaultRuleExceptionListReferences({
+          exceptionLists: request.body.exceptions_list,
+        });
```

Comment:
Do we need to wrap this to catch the possible thrown error and then returning an error with `siemResponse.error`

Reply:
Since this is already within a catch - it would be caught within the route, no?